src/App.css

```diff
@@ -1,59 +1,99 @@
-@import url("https://fonts.googleapis.com/css?family=Kreon|Quicksand");
+@import url('https://fonts.googleapis.com/css?family=Kreon|Quicksand|Pacifico|Lobster|Raleway:600,800');
 
```

Comment:
Font loading takes some time to load, especially fonts that aren't native to the browser. Are these new fonts necessary/integral to the design? If so, then the old ones should be removed. I recommend only 2-3 fonts.